config/settings/common.py

```diff
@@ -301,6 +301,9 @@
     'LOGOUT_URL': '/accounts/logout',
 }
 
+BSYNCR_SERVER_HOST = os.environ.get('BSYNCR_SERVER_HOST')
```

Comment:
NOTE: we need to add a check, probably at the `POST /analyses/` endpoint, that makes sure SEED is configured to use bsyncr